src/Tizen.NUI.Components/Controls/Navigation/Navigator.cs

```diff
@@ -399,6 +404,7 @@ public Page Pop()
                 {
                     //Invoke Page events
                     newTop.InvokeAppeared();
+                    NotifyAccessibilityStatesChangeOfPages(null, newTop);
```

Comment:
I think this line needs to be changed as follows.
```
NotifyAccessibilityStatesChangeOfPages(curTop, newTop);
```

Reply:
done